source/opaque.d.ts

```diff
@@ -4,8 +4,8 @@ export type TagContainer<Token> = {
 	readonly [tag]: Token;
 };
 
-type MultiTagContainer<Token extends PropertyKey> = {
-	readonly [tag]: {[K in Token]: void};
+type Tag<Token extends PropertyKey, TagMetadata> = {
```

Comment:
I decided to rename `MultiTagContainer` to `Tag` because it makes the IDE popups much clearer for tagged types. E.g. in these types:

```ts
type AbsolutePath = Tagged<string, 'AbsolutePath'>;
type NormalizedPath = Tagged<string, 'NormalizedPath'>;
type NormalizedAbsolutePath = AbsolutePath & NormalizedPath;
```

Hovering over `NormalizedAbsolutePath` will now show 

```ts
string & Tag<'AbsolutePath', never> & Tag<'NormalizedPath', never>
```

instead of 

```ts
string & MultiTagContainer<'AbsolutePath', never> & MultiTagContainer<'NormalizedPath', never>
```